src/screens/Member/styles.ts

```diff
@@ -0,0 +1,111 @@
+import {StyleSheet} from "react-native";
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1, justifyContent: 'center'
+    },
+    textInput: {
+        height: 50,
+        borderColor: '#dbdbdb',
+        borderWidth: 1,
+        margin: 10,
+        marginLeft: 20,
+        marginRight: 20,
```

Comment:
`margingLeft` + `marginRight` with the same value can be replaced by a single property `marginHorizontal` like `marginHorizontal:20`